### PR TITLE
Calendar page

### PR DESCRIPTION
**Pull request by RumianR**
**This pull request addresses 2 user stories.**

_US-20: As a student, I want to connect to my google calendar account so that I can manage my calendar events in-app. #23_

Google sign in service was implemented with another service for calendar api to fetch events.

_US-4: As a student, I want the calendar page to be visually appealing and intuitive. #43_

A custom calendar library was used to implement a visually pleasant to look at calendar with a list of events below it.

### CODE REVIEW

File: lib/provider/calendar_notifier.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import 'dart:async';
+import 'package:concordi_around/service/google_calendar.dart';
+import 'package:flutter/widgets.dart';
+
+class CalendarNotifier with ChangeNotifier {
+  var calendarService = new CalendarService();
```
Comment (ooms97) [inline lib/provider/calendar_notifier.dart:6]: remove new keyword

File: lib/service/google_signin.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      }
+    }
+
+  }
+
+  Future<void> _handleSignIn() async {
+    try {
+      await _googleSignIn.signIn();
+    } catch (error) {
+      print(error);
```
Comment (ooms97) [inline lib/service/google_signin.dart:51]: remove print statement

File: lib/provider/calendar_notifier.dart (hunk 1 above)
Comment (RumianR) [reply lib/provider/calendar_notifier.dart]: done

File: lib/service/google_signin.dart (hunk 2 above)
Comment (RumianR) [reply lib/service/google_signin.dart:50]: done

**Pull request merged**